### DISCUSSION TITLE
Is is possible to have multiple people connected at the same time?

### DISCUSSION THREAD

**Discussion by mdeslippe** [wg-easy/wg-easy#453] [Q&A]
I'm running an instance in docker, and when one person is connected, no on else can connect. Is there a configuration to allow multiple connections?

**Comment by awloelxf** [comment]
I don't have this issue. Multiple connection work ok. May be some host restrictions?

**Reply by mdeslippe** [reply]
Took a bit of messing around, seems like WireGuard doesn't work if multiple clients have their kill switch enabled.

**Reply by pheiduck** [reply]
Please provide more Infomation.
Which Version do you use latest, 9,8,7...

**Comment by vithurshanselvarajah** [comment]
Just confirming. Each user has their own configuration correct? 

If you only setup one config / device on the wg-easy portal. Only a single person can access it at a time.
